data/transactions/logic/eval.go

```diff
@@ -4043,6 +4074,10 @@ func opTxBegin(cx *EvalContext) {
 		cx.err = errors.New("itxn_begin without itxn_submit")
 		return
 	}
+	if cx.Proto.IsolateClearState && cx.Txn.Txn.OnCompletion == transactions.ClearStateOC {
+		cx.err = errors.New("clear state prorgams can not issue inner transactions")
```

Comment:
@jannotti Minor - Looks like a typo in the error message.

```suggestion
		cx.err = errors.New("clear state programs can not issue inner transactions")
```